bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3870,12 +3866,15 @@ void ReplicaImp::executeRequestsAndSendResponses(PrePrepareMsg *ppMsg,
     }
     SCOPED_MDC_CID(req.getCid());
     NodeIdType clientId = req.clientProxyId();
+
     accumulatedRequests.push_back(IRequestsHandler::ExecutionRequest{
         clientId,
         static_cast<uint64_t>(lastExecutedSeqNum + 1),
         req.flags(),
-        std::string(req.requestBuf(), req.requestLength()),
-        std::string(ReplicaConfig::instance().getmaxReplyMessageSize() - sizeof(ClientReplyMsgHeader), 0),
+        req.requestLength(),
+        req.requestBuf(),
+        static_cast<uint32_t>(ReplicaConfig::instance().getmaxReplyMessageSize() - sizeof(ClientReplyMsgHeader)),
```

Comment:
Shouldn’t this also be config_.getmaxReplyMessageSize() ?

Reply:
Done.